environment.yml

```diff
@@ -5,57 +5,66 @@ channels:
   - plotly
   - defaults
 dependencies:
+  # mdanalysis dependencies
   - cython
-  - distributed==2021.1.0
-  - dask==2021.1.0
-  - python-graphviz
-  - pytest
-  - decorator
-  - docutils
-  - graphviz
-  - ipywidgets<8.0.0
-  - ipython
-  - ipygany
-  - jedi
-  - jinja2
-  - jupyter_contrib_nbextensions
   - matplotlib
-  - nbclient
-  - nbconvert
-  - nbformat
-  - nbsphinx
   - networkx
-  - nglview
-  - notebook
   - numpy
+  - openmm>=7.0.0
   - packaging
   - parmed
-  - parso
-  - pip
-  - plotly
-  - pybtex
-  - python=3.9
-  - pyvista
-  - scikit-image
   - scikit-learn>=1.1.2
+  - threadpoolctl
+  # building documentation
+  - docutils
+  - pandoc
+  - pybtex
   - sphinx
   - sphinxcontrib-bibtex
-  - threadpoolctl
+  # used by custom_parallel_analysis.ipynb
+  - graphviz
   - tabulate
-  - widgetsnbextension
-  - ipycytoscape>=1.3.0
+  - python-graphviz
+  # used in multiple notebooks
+  - jupyter_contrib_nbextensions
+  - nbconvert
+  - nbformat
+  - nglview
+  - notebook<7
   - seaborn>=0.12.1
-  - openmm>=7.0.0
-  - pandoc
+  # used in pairwise_rmsd.ipynb
+  - plotly
```

Comment:
As we edit or update notebooks, I can see the notebook-specific labels getting outdated quite easily. Instead, could we maybe organise these by purpose? e.g. `plotly`, `ipygany`, `pyvista`, `scikit-image`, `seaborn` are for visualising/plotting; `moviepy` and `nglview` are for visualising/making movies of molecules; `dask` and `distributed` you could switch the heading to `# parallelization`.

Reply:
Changed the grouping here as well, easy to apply